lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -425,24 +434,50 @@ def create_coefficients_cube(
             raise ValueError(msg)
 
         coefficient_index = iris.coords.DimCoord(
-            np.arange(len(optimised_coeffs)),
+            list(range(len(optimised_coeffs))),
             long_name="coefficient_index", units="1")
         coefficient_name = iris.coords.AuxCoord(
             coeff_names, long_name="coefficient_name", units="no_unit")
         dim_coords_and_dims = [(coefficient_index, 0)]
         aux_coords_and_dims = [(coefficient_name, 0)]
-        for coord_name in (
-                ["time", "forecast_period", "forecast_reference_time"]):
-            try:
-                aux_coords_and_dims.append(
-                    (current_forecast.coord(coord_name), None))
-            except CoordinateNotFoundError:
-                pass
-        attributes = {"diagnostic_standard_name": current_forecast.name()}
-        for attribute in current_forecast.attributes.keys():
+
+        # Create a forecast_reference_time coordinate.
+        frt_point = cycletime_to_datetime(self.current_cycle)
+
+        try:
+            frt_coord = (
```

Comment:
What's the benefit of only copying if the coordinate is on the historic cube? Do we want to add these coordinates anyway?

Reply:
Ignore this comment for now.